hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -137,12 +137,22 @@
   private Text dtService;
   private final boolean topologyAwareReadEnabled;
 
+  /**
+   * Creates RpcClient instance with the given configuration.
+   * @param conf Configuration
+   * @throws IOException
+   */
+  public RpcClient(Configuration conf) throws IOException {
```

Comment:
Can we mark this @VisibleForTesting

Reply:
@bharatviswa504 I believe the `VisibleForTesting` annotation doesn't hide the constructor from other code.

Reply:
My comment is for add to a notion of @VisibleForTesting, and I think this can also be removed, as it is used only for testing. And it can be completely removed, and use the new constructor.

Reply:
Sure let's go with removing this old constructor then.